### PR TITLE
MRG: Fix Xdawn example using EpochsVectorizer and y.ravel()

### PR DESCRIPTION
**Pull request by kingjr**
closes https://github.com/mne-tools/mne-python/issues/3433

@kaichogami in the future please check that your new examples are working. There were two bugs in this one.

Also note that you'll have to change the missing `EpochsVectorizer` by a `Vectorizer` in #3409

### CODE REVIEW

**Comment by agramfort** [comment]
travis failure seems legit

File: examples/decoding/plot_decoding_xdawn_transformer.py [hunk 1]
```diff
@@ -40,9 +40,10 @@
                 add_eeg_ref=False, verbose=False)
 
 X = epochs.get_data()
-y = label_binarize(epochs.events[:, 2], classes=[1, 3])
+y = label_binarize(epochs.events[:, 2], classes=[1, 3]).ravel()
 
 clf = make_pipeline(XdawnTransformer(n_components=3),
+                    Vectorizer(),
```
Comment (Eric89GXL) [inline examples/decoding/plot_decoding_xdawn_transformer.py]: Travis says this isn't imported anywhere:

https://travis-ci.org/mne-tools/mne-python/jobs/146206895#L2969

Does this run locally?

Comment (agramfort) [inline examples/decoding/plot_decoding_xdawn_transformer.py]: travis is correct

https://github.com/kingjr/mne-python/blob/51be050157af289225b78ed967cd957a14493ad8/examples/decoding/plot_decoding_xdawn_transformer.py

Comment (kingjr) [inline examples/decoding/plot_decoding_xdawn_transformer.py]: Hadn't saved the modif sorry

**Comment by agramfort** [comment]
merge when travis is happy

**Comment by kingjr** [comment]
@Eric89GXL ready on my end.

**Comment by Eric89GXL** [comment]
FYI the example could still use some tweaking to look better:

https://1795-1301584-gh.circle-artifacts.com/0//home/ubuntu/mne-python/doc/_build/html/auto_examples/decoding/plot_decoding_xdawn_transformer.html

But it should make `CircleCI` happy. Thanks for the fix

**Pull request merged**